app/templates/partials/answers/currency.html

```diff
@@ -9,18 +9,25 @@
 
     <div class="input-type input-type--currency">
 
-      <span class="input-type__type" id="{{answer.id}}-type">£</span>
+      <span class="input-type__type" id="{{answer.id}}-type">{{ get_currency_symbol(answer.currency) }}</span>
+
+        {% if form.errors or form.question_errors %}
+            {% set data_input = form.get_data(answer['id'])|e %}
```

Comment:
Why are we having to escape here? Ideally you shouldn't have to be worrying about it at this point.

Reply:
This could be a legacy code and/or there for a very good reason. To be investigated tomorrow.

Reply:
removed manual escaping from html templates